console/cutil/handler.go

```diff
@@ -0,0 +1,839 @@
+/*
+ * MIT License
+ * Copyright (c) 2016 Samsara Networks Inc.
+ * Modifications copyright 2019 The ChubaoFS Authors.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ */
+
+package cutil
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/chubaofs/chubaofs/proto"
+	client2 "github.com/chubaofs/chubaofs/sdk/graphql/client"
```

Comment:
use graphqlClient replace client 2

Reply:
It is recommended to name it like `gqlclient`.